src/components/tools/table-tool/table-tool.tsx

```diff
@@ -22,6 +22,7 @@ interface IProps {
 
 // all properties are optional
 interface IPartialState {
+  metadata?: TableMetadataModelType;
```

Comment:
Remind me what checks we use to decide what goes in content and what goes in metadata?

Reply:
The metadata is persistent in the sense that it doesn't get blown away/replaced like the content does when the document changes. In this case, `ag-grid` was calling the `indexValueGetter()` between the time when the content was blown away and it was replaced in state. Given that the value it cares about is actually maintained by the metadata, it made sense for it to use it directly rather than trying to go through the volatile content as intermediary.